src/main/java/com/craftaro/epicfarming/database/migrations/_2_subsequentMigration.java

```diff
@@ -13,13 +16,73 @@ public _2_subsequentMigration() {
 
     @Override
     public void migrate(Connection connection, String tablePrefix) throws SQLException {
+        // First, update only rows with invalid Base64 data.
+        fixInvalidData(connection, tablePrefix);
+
+        // Then, alter the column to TEXT.
         try (Statement statement = connection.createStatement()) {
             try {
                 statement.execute("ALTER TABLE " + tablePrefix + "items ALTER COLUMN item SET DATA TYPE TEXT");
             } catch (SQLException e) {
-                System.err.println("H2 syntax failed, trying with MySQL...");
                 statement.execute("ALTER TABLE " + tablePrefix + "items MODIFY COLUMN item TEXT");
             }
         }
     }
+
+    /**
+     * Iterates over all rows in the items table and replaces any invalid Base64 data with an empty string.
+     * Only rows where the data is invalid will be updated.
+     */
+    private void fixInvalidData(Connection connection, String tablePrefix) throws SQLException {
+        if (hasIdColumn(connection, tablePrefix)) {
+            String selectSQL = "SELECT id, item FROM " + tablePrefix + "items";
```

Comment:
Not sure how big this table can get on large servers... But this might cause an OutOfMemory issue for them.

Also, servers that already ran this migration (not sure how old this is and whether we already released a version with it?), won't run it again and get this fix executed.
But maybe that's expected?